src/v2/models/AppState.ts

```diff
@@ -314,26 +315,31 @@ export default class AppState extends Model {
        * expiresAt will be different for each response, hence compare objects without that property
        */
       reRender = false;
+      if (this.get('currentFormName') === 'poll') {
+        /**
+         * returns true: We want to force reRender when currentForm is poll because request has to reinitiate
+         * based on new refresh and UI has to reflect new timer.
+         * We dont technical poll here we just make a request after the specified refresh time each time
+         * we get a new response.
+         */
+        reRender = true;
+      } else if (FORMS_WITH_STATIC_BACK_LINK.includes(this.get('currentFormName'))) {
+        /**
+         * returns true: We want to force reRender if you go back to selection screen from challenge or enroll screen
+         * and re-select the same authenticator for challenge. In this case also new response will be identical
+         * to the old response.
+         */
+        reRender = true;
+      } else if (this.containsMessageWithI18nKey(UNKNOWN_USER_I8N_KEY)) {
```

Comment:
Is this the only new case that needs to be re-rendered? i.e., is the `idx.unknown.user` response actually special or is it just the one that was noticed/reported?

Reply:
It's special: https://okta.slack.com/archives/CCA77QVL2/p1660695611393009.
ftr, more info on this PR for future devs: https://okta.slack.com/archives/CAZH2MWEL/p1660675971972929